apps/appclient/mattermost_client.go

```diff
@@ -170,8 +175,6 @@ func (c *Client) Call(creq apps.CallRequest) (*apps.CallResponse, error) {
 }
 
 func (c *Client) CreatePost(post *model.Post) (*model.Post, error) {
-	post.UserId = c.userID
```

Comment:
:+1: I figured this wasn't required because it's going through the API with token